### PR TITLE
Add widgets for total number of imports and module count

### PR DESCRIPTION
**Pull request by felixLam**
bildschirmfoto 2017-03-14 um 01 41 43

### CODE REVIEW

File: src/Client/Graph/Widgets.elm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+countModules : List Node -> Html msg
+countModules nodes =
+    List.length nodes
+        |> widget "Total modules" "fa-cube" "green"
+
+
+widget : String -> String -> String -> a -> Html msg
+widget title icon color value =
```
Comment (stil4m) [inline src/Client/Graph/Widgets.elm]: Would make the `color` the first argument, then we can nicely add `successWidget = widget "green"`

File: src/Client/Graph/Widgets.elm [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+countImports : List Edge -> Html msg
+countImports edges =
+    List.length edges
+        |> widget "Total imports" "fa-arrow-circle-o-down" "green"
+
+
+countModules : List Node -> Html msg
+countModules nodes =
+    List.length nodes
+        |> widget "Total modules" "fa-cube" "green"
```
Comment (stil4m) [inline src/Client/Graph/Widgets.elm]: These functions belong to the caller module. Will make it easier to reuse this module in the future

File: src/Client/Graph/Widgets.elm (hunk 1 above)
Comment (felixLam) [inline src/Client/Graph/Widgets.elm]: will resort and use a more descriptive type

File: src/Client/Graph/Widgets.elm (hunk 2 above)
Comment (felixLam) [inline src/Client/Graph/Widgets.elm]: make sense

Comment (felixLam) [inline src/Client/Graph/Widgets.elm]: Will move the Widget to Client.View

**Pull request merged**